SpywareFilter/sections/cookies.txt

```diff
@@ -34,4 +33,142 @@
 $cookie=_parsely_visitor
 $cookie=_parsely_session
 ! Tealium Universal tag
-$cookie=utag_main
\ No newline at end of file
+$cookie=utag_main
+! Google - https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage
```

Comment:
it would be really cool if we could add a comment to every cookie (where this is possible).

The thing is that those pages you're linking are changing with time and we'll simply lose this info if we don't record it here.

Reply:
Some cookies with description listed in tables:
https://www.staqq.co/cookie-policy
https://support.devex.com/hc/en-us/articles/115000959453-Devex-Privacy-Statement

Reply:
Added Reblaze and tinypass cookies and some comments.